lib/config/scheme.js

```diff
@@ -0,0 +1,24 @@
+const configParser = require('gemini-configparser');
+
+const root = configParser.root;
+const section = configParser.section;
+const option = configParser.option;
+
+const shouldBeString = require('../metadata/validation-utils').shouldBeString;
+
+const ENV_PREFIX = `${require('../../package').name}_`;
+
+module.exports = root(
+    section({
+        username: option({
+            validate: shouldBeString
+        }),
+
+        apiToken: option({
+            validate: shouldBeString
+        }),
+
+        url: option({
+            validate: shouldBeString
+        })
+    }), {envPrefix: ENV_PREFIX});
```

Comment:
`envPrefix` isn't currently in the gemini-configparser documentation, so I was confused on why it's here. Maybe add a comment explaining it? Or just use the documented feature `parseEnv` instead.